packages/docs/developer-resources/contractkit/odis.md

```diff
@@ -0,0 +1,113 @@
+# Calling Oblivious Decentralized Identifier Service (ODIS)
+This guide walks through using Contract Kit to query the on-chain identifier given a phone number or other identifier (ex. Twitter handle). See [this overview document](../../celo-codebase/protocol/identity/phone-number-privacy.md) for more details on ODIS. One of Celo's key features is the ability to tie a phone number to a Celo address. This provides a convenient payment experience for Celo users. To map a phone number to an address, the on-chain identifier for a given phone number must first be retrieved. With this identifier, the address can be looked up on-chain.
```

Comment:
I worry that mentioning Twitter implies we support other attestation types. Atm we don't